lib/src/base-styles/_nx-btn.scss

```diff
@@ -31,7 +31,7 @@
   }
 }
 
-.nx-btn {
+.nx-btn, .nx-btn[href] {
```

Comment:
Was there a reason to use `.nx-btn[href]` rather than `a.nx-btn`?  The latter seems cleaner and likely more performant to me.

Reply:
> Was there a reason to use `.nx-btn[href]` rather than `a.nx-btn`? The latter seems cleaner and likely more performant to me.

No reasoning for using one over the other, so changed here: [eb5d38c](https://github.com/sonatype/sonatype-react-shared-components/pull/1137/commits/eb5d38c1ad7af033d1a7a25f66d6d734df3212ee)